interview-questions.md

```diff
@@ -7,55 +7,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer:git is a open sourced version control system used by programmers.
+  git is the tool used to track and manage your code source history and github is the cloud base hosting service that lets you manage git repositories.
 
-  Researched answer:
+  Researched answer: Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories. If you have open-source projects that use Git, then GitHub is designed to help you better manage them
```

Comment:
Hey I found the stack overflow that this answer came from
```
Simply put, Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories. If you have open-source projects that use Git, then GitHub is designed to help you better manage them.
```

Reply:
https://stackoverflow.com/questions/13321556/difference-between-git-and-github

Reply:
Git is a system that helps us manage, rack and locate open projects in repositories inside a cloud based host, which is GitHub.